ztrouy/kneel-diamonds-python
# Feature - Create Orders

### Purpose
To allow Users to create a new Order

### Files Changed
- Added method that adds a new Order to the database in `order_views.py`
- Imported create_order method in the `__init__.py` dunder init module in the `/views` directory
- Added POST endpoint for Orders that creates a new Order in `json-server.py`
- Removed `kneeldiamonds.sqlite3`
- Renamed `kneeldiamonds.sql` to `db.sql`

### To Test
Fetch, recreate database, and run `json-server.py`, then confirm the following
- POST endpoint `/orders` creates a new Order when provided a valid Order object
